handler_error.go

```diff
@@ -87,6 +87,12 @@ func (e *ErrorHandler) HandleError(w http.ResponseWriter, r *http.Request, errMs
 
 		t := time.Now()
 
+		if ctxGetDefaultVersion(r) {
+
+			vinfo := ctxGetVersionInfo(r).Name
+			w.Header().Set("X-Tyk-Default-Version", vinfo)
```

Comment:
I see string value `X-Tyk-Default-Version` repeated several times, might worth to introduce a constant for it